src/main/java/ch/puzzle/eft/model/ExamFileModel.java

```diff
@@ -15,6 +15,12 @@ public String getSubjectName() {
                 .getName();
     }
 
+    public String getUrl() {
```

Comment:
Maybe something like path instead of url would be more fitting, as it references a local resource?